pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling"]
+requires = ["hatchling", "hatch-build-scripts"]
```

Comment:
Great! So I guess that it's not possible to list libsass here?

I've clicked "watching" on the hatch-build-scripts repo since it seems like it's a small project: https://github.com/rmorshea/hatch-build-scripts

Reply:
That's a good observation!!

Based on the PEP documentation, we can place it there (https://peps.python.org/pep-0518/) I could explore that just in case, the only thing I'm not OK is that using this solution will make the lib sass required for everything, since for each environment this is build on dev mode.

Reply:
Not a bad idea. I don't have any strong opinions on this one, if the libsass dependency isn't really problematic and pysassc is fast to run, then it's okay for me. But we can easily just wait and see if the new setup causes any problems, it also seems fine as it is :+1: